common/policy/load_policies.sh

```diff
@@ -38,7 +38,8 @@ conjur variable values add "secrets/var with spaces" "some-secret"
 conjur variable values add "secrets/var+with+pluses" "some-secret"
 
 # populate Conjur secrets for the demo
-conjur variable values add secrets/db_username "$POSTGRES_USERNAME"
-conjur variable values add secrets/db_password "$POSTGRES_PASSWORD"
+# TODO figure out why $POSTGRES_USERNAME / $POSTGRES_PASSWORD are empty
+#conjur variable values add secrets/db_username "$POSTGRES_USERNAME"
+#conjur variable values add secrets/db_password "$POSTGRES_PASSWORD"
```

Comment:
don't we need these? whey are they used?

Reply:
demo demo demo